silkworm/db/datastore/snapshots/elias_fano/elias_fano_decoder.hpp

```diff
@@ -26,9 +26,15 @@ struct EliasFanoDecoder : public snapshots::Decoder {
 
     ~EliasFanoDecoder() override = default;
 
-    void decode_word(Bytes& word) override {
-        value = EliasFanoList32::from_encoded_data(word);
-        value.data_holder = std::move(word);
+    void decode_word(Word& word) override {
+        if (word.holds_bytes()) {
+            auto word_value = std::get<Bytes>(word);
```

Comment:
```suggestion
            Bytes& word_value = std::get<Bytes>(word);
```

Reply:
I've refactored this to avoid using l-values altogether